src/schema/v2/sale/index.ts

```diff
@@ -234,6 +237,62 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      auctionsDetailFormattedStartDateTime: {
+        type: GraphQLString,
+        description:
+          "A more granular formatted description of when the auction starts or ends or if it has ended",
+        resolve: ({ start_at, end_at, ended_at }, _options) => {
+          return auctionsDetailFormattedStartDateTime(
+            start_at,
+            end_at,
+            ended_at
+          )
+        },
+      },
+      auctionsDetailFormattedStartEndSignal: {
+        type: GraphQLString,
+        description:
+          "A colored signal to users indicating how much time until sale opens/closes",
+        resolve: (
+          { start_at, end_at, ended_at },
+          _options,
+          { defaultTimezone }
+        ) => {
+          return auctionsDetailFormattedStartEndSignal(
+            start_at,
+            end_at,
+            ended_at,
+            defaultTimezone
+          )
+        },
+      },
+      auctionsDetailStartEndSignalColor: {
+        type: GraphQLString,
+        description:
+          "The color for the signal that indicates how much time until a sale opens/closes",
+        resolve: ({ ended_at, end_at }, _options, { defaultTimezone }) => {
+          return auctionsDetailStartEndSignalColor(
+            ended_at,
+            end_at,
+            defaultTimezone
+          )
+        },
+      },
+      auctionsDetailCascadingIntervalLabel: {
+        type: GraphQLString,
+        description:
+          "A label indicating the interval in minutes in which lots close",
+        resolve: ({ ended_at, cascading_end_time_interval }, _options) => {
+          if (cascading_end_time_interval === null) return ""
+          if (ended_at) {
```

Comment:
I think we'd probably prefer to return null if we don't want anything back from this

Reply:
gotcha I thought about that, but wasn't sure since the specified type is GraphQLString

Reply:
All types are nullable unless they're modified with `NonNull` (`!`). Give a look over the GraphQL docs sometime!